### ISSUE TITLE
Only check inside the folder

### ISSUE THREAD

**Issue by DankMemeGuy** [ermig1979/AntiDupl#91]
I have a vast bulk of folders that I need to delete duplicates for. However, it should only check within the folder. So if I put in folders: 1,2,3,4 and Folders 1 and 3 have the same file, BUT within folders 1 and 3 there is only 1 file (no duplicates) then nothing should be removed. But if folder 4 has 2 copies of the same file then it would remove the duplicate.

File names are definitely not identical, but the size should be the same since they are identical images, not sure about MD5 hash.

AntiDupl seems to check all the folders when put in the program, instead of just the folder itself and its subfolders. Would be a nice feature to have.

**Comment by yfdyh000** [comment]
Check the "Compare images inside one directory" and uncheck the "Compare images from one path of search with one another" options.
https://github.com/ermig1979/AntiDupl/blob/286bf8405ee05066b27cc068dd0542eff818cf3f/docs/data/help/english/options.html#L89

**Comment by DankMemeGuy** [comment]
I had actually tried that option but it didn't work so I figured I was misunderstanding it. Maybe the changes didn't get saved when I selected it!

**Comment by DankMemeGuy** [comment]
Nope, still didn't work. I put two search paths and they both still looked into each other instead of saying within their own folder:

https://i.imgur.com/PFM6Xoi.png

**Comment by DankMemeGuy** [comment]
Bump?

**Comment by DankMemeGuy** [comment]
If someone is willing to implement this feature, I can compensate them something.

**Comment by heinrichI** [comment]
А что если каждую директорию добавить по отдельности? Тогда работает как хотите? И вообще я не очень понял в чем проблема. Может переформулируете или другой пример приведете?

**Comment by DankMemeGuy** [comment]
> А что если каждую директорию добавить по отдельности? Тогда работает как хотите? И вообще я не очень понял в чем проблема. Может переформулируете или другой пример приведете?

I have a few thousand directories, I drag and drop the directories into AntiDupl. Within each directory there are some duplicates, these should be removed. BUT, in other directories I have same image, BUT those should remain.

Example: (ignore file names for this example, only the content of the image is the duplicate, the file name is irrelevant)

Folder1/image.jpg
Folder1/image2.jpg (same as Folder2/image.jpg)
Folder1/image3.jpg (same as Folder1/image4.jpg)
Folder1/image4.jpg

Folder2/image.jpg
Folder2/image2.jpg
Folder2/image.3jpg (same as Folder2/image.jpg)
Folder2/image4.jpg

When I put in Folder1 and Folder2 into AntiDupl, the only images that should be flagged as duplicate are: 

Folder2/image.3jpg (same as Folder2/image.jpg)
Folder1/image3.jpg (same as Folder1/image4.jpg)

The problem is that AntiDupl also considers the following a duplcaite:
Folder1/image2.jpg (same as Folder2/image.jpg)

**Comment by DankMemeGuy** [comment]
Anyone?
